docs/docs-beta/src/components/PyObject.tsx

```diff
@@ -3,38 +3,53 @@ import React from 'react';
 
 export const SearchIndexContext = React.createContext(null);
 
+/**
+ * PyObject component renders a formatted link to the Python API docs.
+ *
+ * Because we are using the `<Link>` component, Docusaurus will validate broken links on build.
+ */
 export const PyObject: React.FunctionComponent<{
+  section: string;
   module: string;
   object: string;
   method?: string;
   displayText?: string;
   pluralize?: boolean;
   decorator?: boolean;
-}> = ({object, method, displayText, pluralize = false, decorator = false}) => {
+}> = ({
+  section,
+  method,
+  object,
+  displayText,
+  module = 'dagster',
+  pluralize = false,
+  decorator = false,
+}) => {
   let textValue = displayText || object;
   if (pluralize) {
     textValue += 's';
   }
   if (decorator) {
-    textValue = '@' + textValue;
+    if (module === 'dagster') {
+      textValue = '@dg.' + textValue;
+    } else {
+      textValue = '@' + module + '.' + textValue;
+    }
   }
   if (method) {
     textValue += '.' + method;
   }
 
-  const handleClick = (e: React.MouseEvent) => {
-    e.preventDefault();
-    alert('PyObject not implemented yet');
-  };
+  // As we don't have access to the searchContext like we did in the Next.js version of docs, we
```

Comment:
Could you add this to the migration section of the CONTRIBUTING doc? Sounds like we can just copy/paste `<PyObject>` from existing docs and add the section.

Reply:
Ah yes, good idea.

Reply:
Updated!